integration-templates/cal-com-v2/nango.yaml

```diff
@@ -0,0 +1,163 @@
+                endpoint: GET /event/types
+                output: EventType
+                description: Retrieve all event types per a user
+
+models:
+    EventType:
+        id: number
+        teamId: null | number
+        schedulingType: null | string
+        userId: number
+        metadata:
+            ___string: any
+        description: string | null
+        hidden: boolean
+        slug: string
+        length: number
+        title: string
+        requiresConfirmation: boolean
+        position: number
+        offsetStart: number
+        profileId: string | null
+        eventName: string | null
+        parentId: number | null
+        timeZone: string | null
+        periodType: string
+        periodStartDate: string | null
+        periodEndDate: string | null
+        periodDays: number | null
+        periodCountCalendarDays: number | null
+        lockTimeZoneToggleOnBookingPage: boolean
+        requiresBookerEmailVerification: boolean
+        disableGuests: boolean
+        hideCalendarNotes: boolean
+        minimumBookingNotice: number
+        beforeEventBuffer: number
+        afterEventBuffer: number
+        seatsPerTimeSlot: number | null
+        onlyShowFirstAvailableSlot: boolean
+        seatsShowAttendees: boolean
+        seatsShowAvailabilityCount: boolean
+        scheduleId: number | null
+        price: number
+        currency: string
+        slotInterval: number | null,
+        successRedirectUrl: string | null
+        isInstantEvent: boolean
+        aiPhoneCallConfig: string | null
+        assignAllTeamMembers: boolean
+        recurringEvent: boolean | null
+        locations:
+            - type: string
+        bookingFields: string | null
+        useEventTypeDestinationCalendarEmail: boolean
+        secondaryEmailId: string | null
+        bookingLimits: boolean | null
+        durationLimits: boolean | null
+        hashedLink: string | null
+        children:
+            - __string: any
+        hosts:
+            - __string: any
+        userIds: number[]
+    User:
+        username: string
+        name: string
+        weekStart: string
+        organizationId: number
+        avatarUrl: string
+        profile:
+            username: string
+            id: number
+            userId: number
+            uid: string
+            name: string
+            organizationId: number
+            organization:
+                id: number
+                slug: string
+                name: string
+                metadata:
+                    ___string: any
+            upId: string
+            image: string
+            brandColor: string
+            darkBrandColor: string
+            theme: string
+            bookerLayouts:
+                ___string: any
+        bookerUrl: string
+    Attendee:
+        id: number
+        email: string
+        name: string
+        timeZone: string
+        locale: string
+        bookingId: number
+    Event:
+        id: number
```

Comment:
id should be string

Reply:
Yeah, for Nango purposes it should be a string -- updated